### PR TITLE
extend image singlespa/import-map-deployer and use config.json file

### PR DESCRIPTION
**Pull request by kristianmandrup**
Is this a correct sample setup to work correctly with `singlespa/import-map-deployer`?

### CODE REVIEW

File: conf.js [hunk 1]
```diff
@@ -1,11 +0,0 @@
-module.exports = {
-  username: process.env.HTTP_USERNAME,
```
Comment (internettrans) [inline conf.js]: This will break things for the import map deployer running on https://microfrontends.app. Why did you delete it? A `conf.js` file works fine - if you undo your changes here I'll merge in the rest of the PR

Comment (kristianmandrup) [reply conf.js]: I thought it had to be a json file. Does it support both json and js config file?
Wasn't entirely clear. I'll fix it today.

**Comment by kristianmandrup** [comment]
I've now reverted the `conf.js` file, added a Readme and typings for NodeJS (to avoid red scribbles in VSCode).

File: tsconfig.json [hunk 2]
```diff
@@ -0,0 +1,66 @@
+{
```
Comment (internettrans) [inline tsconfig.json]: Why is this file here even though this project doesn't use typescript?

Let's remove it.

**Comment by kristianmandrup** [comment]
Yes, not easential. My VS code setup was complaining about the global `process` refs without it though.

**Review by internettrans — APPROVED** [review]
Thanks
